src/reactpy/types.py

```diff
@@ -885,6 +885,10 @@ class JsonImportSource(TypedDict):
     fallback: Any
 
 
+class JavaScript(str):
```

Comment:
Add a docstring to explain that this class is a simple way of marking JavaScript code to be executed in by the browser